deps/rabbitmq_shovel/src/rabbit_amqp091_shovel.erl

```diff
@@ -251,6 +254,9 @@ handle_dest(#'basic.cancel'{}, #{name := Name}) ->
 
 handle_dest({'EXIT', Conn, Reason}, #{dest := #{current := {Conn, _, _}}}) ->
     {stop, {outbound_conn_died, Reason}};
+    
+handle_dest({'EXIT', _Pid, {shutdown, {server_initiated_close, ?PRECONDITION_FAILED, Reason}}}, _State) ->
+    {stop, {outbound_consumer_no_ack, Reason}};
```

Comment:
Just like above, it should be `destination_precondition_failed` or something generic like that.